src/Components/Carousel/Carousel.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import './Carousel.css';
+import Card from '../Card/Card';
+import Carousel from 'react-multi-carousel';
+import 'react-multi-carousel/lib/styles.css';
+
+const MyCarousel = ({ movies, badge }) => {
+  const cardElements = movies.map((movie) => (
+    <Card
+      averageRating={movie.averageRating}
+      backdropPath={movie.backdrop_path}
+      id={movie.id}
+      key={movie.id}
+      releaseDate={movie.release_date}
+      title={movie.title}
+    />
+  ));
+
+  const responsive = {
+    desktop: {
+      breakpoint: { max: 3000, min: 1024 },
+      items: 6,
+    },
+    tablet: {
+      breakpoint: { max: 1024, min: 464 },
+      items: 3,
+    },
+    mobile: {
+      breakpoint: { max: 464, min: 0 },
+      items: 1,
+    },
+  };
+
+  const isMobile = window.innerWidth <= 464;
+
+  return (
+    <div>
+      <h2 tabindex='0'>{badge} Movies</h2>
+      <Carousel
```

Comment:
So many props!! Third party libraries is like learning a whole new technology sometimes!